web/src/SidebarResources.tsx

```diff
@@ -94,9 +193,14 @@ function SidebarGroupedByLabels(props: SidebarProps) {
 
         labelsToResources[label].push(item)
       })
-    } else {
+    } else if (!item.isTiltfile) {
       unlabeled.push(item)
     }
+
+    // Display the Tiltfile outside of the label groups
```

Comment:
i wonder if the Tiltfile itself should have a special built-in label...in the future, we might have more than one tiltfile resource

Reply:
this is definitely something to keep in mind! i think right now, the label grouping code can handle multiple tiltfiles, as long as the `isTiltfile` property is set properly further up stream.

i'm curious, from a technical standpoint, is it possible for a user to add labels to a tiltfile from the `Tiltfile`? 👀